Lib/fontbakery/profiles/universal.py

```diff
@@ -2207,5 +2208,69 @@ def com_google_fonts_check_linegaps(ttFont):
         yield PASS, "OS/2 sTypoLineGap and hhea lineGap are both 0."
 
 
+@check(
+    id = 'com.google.fonts/check/STAT_in_statics',
+    conditions = ['not is_variable_font'],
```

Comment:
Using a `has_STAT_table` condition gives us a SKIP.
The way I implemented it, results in a PASS, which is also valid for this case.

It is a tricky subtle difference. In this case I though it would be reasonable to say that a static font without a STAT table passed the check. But a SKIP would also be kind of OK, meaning that "this check does not apply to this font, skip it".

Reply:
If you use the `has_STAT_table` condition you don't event have to have `if "STAT" in ttFont:` within the check.
And a SKIP result for a font that doesn't even have a STAT table is the correct outcome IMO.

Reply:
Sounds good either way.